kratos/includes/global_pointer.h

```diff
@@ -63,10 +56,7 @@ class GlobalPointer {
    */
   GlobalPointer(boost::shared_ptr<TDataType> DataPointer, int Rank = 0)
```

Comment:
`Kratos:: ...`?

Reply:
Not sure what you mean, something wrong there?

Reply:
nothing wrong, was just wondering why you don't use the new `Kratos::shared_ptr` => #1328

Reply:
:+1:  But I will make the changes in a separate PR, since this is very old and if I merge the master with this it will crash